api/docs/developer-system-initialization.md

```diff
@@ -66,6 +66,13 @@ git secrets --scan /path/to/file (/other/path/to/file *)
 git secrets --scan -r /path/to/directory
 ```
 
+## .npmrc
```

Comment:
I think this is too buried still - please add something around L31 in this file, or create a link from there